pandas/io/parquet.py

```diff
@@ -221,19 +223,20 @@ def read(
         path,
         columns=None,
         use_nullable_dtypes: bool = False,
+        dtype_backend: DtypeBackend | lib.NoDefault = lib.no_default,
         storage_options: StorageOptions = None,
         **kwargs,
     ) -> DataFrame:
         kwargs["use_pandas_metadata"] = True
 
-        dtype_backend = get_option("mode.dtype_backend")
         to_pandas_kwargs = {}
-        if use_nullable_dtypes:
-            if dtype_backend == "pandas":
-                from pandas.io._util import _arrow_dtype_mapping
+        if dtype_backend == "numpy_nullable":
+            from pandas.io._util import _arrow_dtype_mapping
 
-                mapping = _arrow_dtype_mapping()
-                to_pandas_kwargs["types_mapper"] = mapping.get
+            mapping = _arrow_dtype_mapping()
+            to_pandas_kwargs["types_mapper"] = mapping.get
+        elif dtype_backend == "pyarrow":
+            to_pandas_kwargs["types_mapper"] = pd.ArrowDtype  # type: ignore[assignment]  # noqa
```

Comment:
Probably an idea for a future refactoring more than for this PR, but feels like it could make sense to encapsulate most of the backend stuff in a class. Something like:

```
backend = _Backend('pyarrow')  # does the validation
backend.mapping
backend.is_nullable
backend.is_default
...
```

IMO would make code a bit more readable, and also it'd probably make it easy to change things like the default in the future.

Reply:
Sounds like a good idea, but as a follow up as you said. Will look into it when we are through here